package.json

```diff
@@ -53,6 +52,8 @@
     "react-dom": "^16.8.4",
     "react-redux": "^7.1.1",
     "react-router-dom": "^5.0.1",
-    "redux": "^4.0.4"
+    "redux": "^4.0.4",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
Actully I am using _composeWithDevTools_ from this pkg, and if we are using any function from the pakage then we need to add it to the dependencies.

Reply:
`composeWithDevTools` is only used in `development` environment right?

Reply:
Exactly but there was a eslint error so i did this.

Reply:
What was the error?

Reply:
![image](https://user-images.githubusercontent.com/29652551/65422105-2bb4c680-de23-11e9-8d5e-0199b9116cd1.png)

Reply:
![image](https://user-images.githubusercontent.com/29652551/65422132-41c28700-de23-11e9-9534-133925c9f03b.png)

Reply:
Check this https://stackoverflow.com/questions/44939304/eslint-should-be-listed-in-the-projects-dependencies-not-devdependencies

Reply:
fixed https://github.com/DivyanshBatham/cb-candids-client/pull/35/commits/254a53a861f90c66cc09ed928aee1be045f91a98!